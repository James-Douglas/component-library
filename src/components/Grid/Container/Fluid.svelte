<svelte:options tag='ctm-fluid' />
<div class='container'>
  <slot></slot>
</div>

<style>
  @import './container.css';
</style>
