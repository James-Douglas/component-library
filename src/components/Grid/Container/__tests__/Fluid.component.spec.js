import React from 'react';
import { render } from '@testing-library/react';
import Fluid from '../Fluid.component';

describe('Container', () => {
  it('renders correctly without props', () => {
    const { container } = render(<Fluid />);
    expect(container).toMatchSnapshot();
  });

  it('has a container and fixed class by default', () => {
    const { container } = render(<Fluid />);

    const fluid = container.querySelector('.container');

    expect(container).toMatchSnapshot();
    expect(fluid).toBeDefined();
    expect(fluid.classList[1]).toBeUndefined();
  });
});
