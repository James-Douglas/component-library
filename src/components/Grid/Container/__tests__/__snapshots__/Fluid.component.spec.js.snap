// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container has a container and fixed class by default 1`] = `
<div>
  <div
    class="container "
  />
</div>
`;

exports[`Container renders correctly without props 1`] = `
<div>
  <div
    class="container "
  />
</div>
`;
